chart/compass/charts/gateway/templates/oathkeeper-rules.yaml

```diff
@@ -95,6 +95,30 @@ spec:
 ---
 apiVersion: oathkeeper.ory.sh/v1alpha1
 kind: Rule
+metadata:
+  name: compass-broker-certs
```

Comment:
`compass-system-broker-certs` - we'll have other brokers in the future as well, so we need to differentiate them